corpdir-srv-api/src/main/java/com/github/bordertech/corpdir/modify/api/v1/ContactWriteService.java

```diff
@@ -0,0 +1,24 @@
+package com.github.bordertech.corpdir.modify.api.v1;
```

Comment:
To keep the package name consistent with the module, do you think this should be `com.github.bordertech.corpdir.api.v1.modify`?

This also keeps all the packages/classes involved with `v1` under the same package `corpdir.api.v1` instead of `corpdir.api.v1`,  `corpdir.readonly.api.v1` and `corpdir.modify.api.v1`.